codechecker_lib/logger.py

```diff
@@ -41,6 +41,7 @@ def add_verbose_arguments(parser):
 
 # ------------------------------------------------------------------------------
 logging.DEBUG_ANALYZER = 15
+DEBUG_ANALYZER = logging.DEBUG_ANALYZER
 logging.addLevelName(logging.DEBUG_ANALYZER, 'DEBUG_ANALYZER')
```

Comment:
I think we could use the `DEBUG_ANALYZER` instead of `logging.DEBUG_ANALYZER` when we add the new level name.